plugins/wd/wd.plugin.zsh

```diff
@@ -7,9 +7,4 @@
 #
 # @github.com/mfaerevaag/wd
 
-# Handle $0 according to the standard:
```

Comment:
We should not remove this to properly handle `$0`. Could you revert this particular change? For the rest, LGTM!

Reply:
Done!